pulsar-client/src/main/java/org/apache/pulsar/client/impl/ReaderBuilderImpl.java

```diff
@@ -112,6 +112,13 @@ private ReaderBuilderImpl(PulsarClientImpl client, ReaderConfigurationData<T> co
         return this;
     }
 
+    @Override
+    public ReaderBuilder<T> startMessageIdInclusive(MessageId messageId) {
```

Comment:
Since we're now forced to have 2 methods anyways (to not breaking compatibility), I'd prefer to keep the included/non-included as a flag, rather than a different option. eg: 

```
client.newReader()
      .topic("my-topic")
      .startMessageId(myMessageId)
      .startMessageIdIncluded(true)
      .create();
```

One other case to consider is with `MessageId.latest`. There we should be able to read the last 1 message in the topic.

Reply:
> I'd prefer to keep the included/non-included as a flag, rather than a different option

Agree, changing it.

> One other case to consider is with MessageId.latest.

:+1:  I will add a test case for this.

Reply:
Done!